src/testing_system/internal/adapter/retrievers/etl.py

```
@@ -18,10 +22,12 @@ def retrieve(self, request: RetrievalRequest) -> RetrievalResponse:
             "clean": self.clean
         }
         response = requests.get(f"{self.ETL_SERVICE_URL}/search", params=params)
```

Ruff (0.15.13)

[error] Probable use of `requests` call without timeout

(S113)